### DISCUSSION TITLE
SIMPL applied to reanalyze diverse hippocampal data

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#166] [Data reanalysis ideas]
[Really cool paper](https://tomge.org/papers/simpl/) by @TomGeorge1234 shows you can improve receptive field modeling by not using position but instead using a position-initialized latent space learned via EM. See excellent twee-print [here](https://x.com/TomNotGeorge/status/1856712367486210339).

This could be applied to many dandisets, e.g. from [Buzsaki](https://dandiarchive.org/dandiset/search?search=Buzs%C3%A1ki), [Giocomo](https://dandiarchive.org/dandiset/search?search=Giocomo), [Frank](https://dandiarchive.org/dandiset/search?search=Frank), [Moser](https://dandiarchive.org/dandiset/search?search=Moser). There are currently [48 dandisets](https://dandiarchive.org/dandiset/search?search=hippocamp&page=1&sortOption=0&sortDir=-1&showDrafts=true&showEmpty=false) that match for "hippocamp"

I would love to see how this changes the shapes of receptive fields from traditional methods. Maybe we can learn new associations with the stats of receptive fields and brain area, neuron type, inter-neuronal correlations, behavioral mode, etc. etc.

 Message here if you would be interested in this!

**Comment by TomGeorge1234** [comment]
This sounds like a great idea, thanks for raising this as a discussion @bendichter. I'm happy to support anyone who tries it. 

Firstly, the SIMPL code base can be found [here](https://github.com/TomGeorge1234/SIMPL/tree/master) 

The ideal dataset is a large spiking dataset where we could easily get $S$ (a matrix of spike counts shape N x T (neurons by time bins)) and a matched behavioural trajectory $X$ (a matrix of positions, in 2D that would be x and y vs time bins, eg shape (2 x T)). The more time bins and the more neurons, the better. Then [this demo](https://github.com/TomGeorge1234/SIMPL/blob/master/demos/simpl_demo.ipynb) will directly apply.

**Comment by bendichter** [comment]
[Here](https://github.com/dandi/example-notebooks/blob/master/000582/Sargolini2006/000582_Sargolini2006_demo.ipynb)'s an example of a notebook that would be relevant for pulling the appropriate data structures from an NWB file. This is older data with not very many neurons, but might be a good starting point.

**Comment by TomGeorge1234** [comment]
Tagging @colleenjg who has experience working with hippocampal datasets, and Dandi (and me!) in case she has expertise which could help propel this. 

Also @masahironakano who has overlapping interest in SIMPL and reanalysing place field data...
